user_interface/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Rodolfo Peixoto">
+    <meta name="keywords" content="cursos, ruby, flask, python, golang, sinatra, java, springboot" >
+    <meta name="description" content="Curso de desenvolvimento de software ensinando redes e integrando os sistemas" >
+    <title>Course HTML skill.dev</title>
+    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@3.4.6/base.min.css" rel="stylesheet" />
+  </head>
+  <body>
+    <h1 class="text-3xl font-bold underline">
+      Hello world!
+    </h1>
+    <form action="/action_page.php">
+      <input type="radio" id="html" name="fav_language" value="HTML">
+      <label for="html">HTML</label><br>
```

Comment:
[ Sugestão ] 

```suggestion
  <body>
    <h1 class="text-3xl font-bold underline">
    <form action="/action_page.php">
      <label for="html">HTML</label><br>
```

O que você acha de modificar o código acima por um código diferente, porque pode gerar problemas de performance.


**Referência**

https://guides.rubyonrails.org/v3.2/performance_testing.html